packages/runner/README.md

```diff
@@ -8,56 +8,106 @@ Run scanning for vulnerabilities just from your unit tests on CI phase.
+
+const configuration = new Confiruration({ cluster: 'app.neuralegion.com' });
+const runner = new SecRunner(configuration);
+
+// or
+
+const runner2 = new SecRunner({ cluster: 'app.neuralegion.com' });
+```
+
+After that, you have to initialize a `SecRunner` instance:
+
+```ts
+await runner.init();
+```
+
+The runner is now ready to perform your tests, but you have to create a scan.
 
-    - Initialize `SecRunner` instance by calling `init()`.
+To dispose a runner, you just need to call the `clear` method:
 
-3.  Setup `SecScan`:
+```ts
+await runner.clear();
+```
+
+### Starting scan
+
+To start scanning your application, first you have to create a `SecScan` instance, as shown below:
+
+```ts
+const scan = runner.createScan({ tests: [TestType.XSS] });
+```
+
+Below you will find a list of parameters that can be used to configure a `Scan`:
 
-    - Create `SecScan` instance by calling `createScan(opts: SecScanOptions)` of `SecRunner` instance providing at least list of `tests`
+| Option                 | Description                                                                                                                                                                                   |
+| ---------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `tests`                | The list of tests to be performed against the target application. [Learn more about tests](https://docs.brightsec.com/docs/vulnerability-guide)                                               |
+| `smart`                | Determine whether scan is smart or simple. Use automatic smart decisions such as: parameter skipping, detection phases, etc. to minimize scan time. Enabled by default.                       |
+| `skipStaticParams`     | Use an advanced algorithm to automatically determine if a parameter has any effect on the target system's behavior when changed, and skip testing such static parameters. Enabled by default. |
+| `poolSize`             | Sets the maximum concurrent requests for the scan, to control the load on your server. By default, `10`.                                                                                      |
+| `attackParamLocations` | Defines which part of the request to attack. By default, `body`, `query`, and `fragment`.                                                                                                     |
+| `slowEpTimeout`        | Skip entry-points that take longer to respond than specified ms value. By default, 1000ms.                                                                                                    |
+| `targetTimeout`        | Measure timeout responses from the target application globally, and stop the scan if the target is unresponsive for longer than the specified time. By default, 5s.                           |
```

Comment:
`targetTimeout` is not exposed in `SecScanOptions`. should be either added there or removed from readme

Reply:
I guess we should provide user ability to configure it as well